src/Stratis.Bitcoin.Features.BlockStore/Converters/BtcDecimalJsonConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Converters
+{
+    public class BtcDecimalJsonConverter : JsonConverter
+    {
+        private const int MinDecimals = 8;
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
```

Comment:
I think this is equivalent to d.ToString("0.00000000###########") or something like that ? not sure how many "#" you need, but maybe it can be simplified this way

Reply:
[Fact]
        public void BtcDecimalFormattingTest()
        {
            var converter = new BtcDecimalJsonConverter();
            using(var sb = new StringWriter())
            using (var writer = new JsonTextWriter(sb))
            {
                var jsonSerializer = JsonSerializer.Create();
                decimal value = (decimal).200;
                converter.WriteJson(writer, value, jsonSerializer);
                var result = sb.GetStringBuilder().ToString();
                result.Should().Be(value.ToString("0.00000000#######"));
            }
        } 

That could be used to test my suggestion is correct , I did a few tests and think I am right but feel free to do more, or let me know if I got confused